cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/MyUserDTO.java

```diff
@@ -17,6 +17,7 @@
 import javax.validation.constraints.Pattern;
 import javax.validation.constraints.Size;
 import java.util.Date;
+import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AUselessImportCheck)